artemis-core-client/src/main/java/org/apache/activemq/artemis/core/remoting/impl/ssl/SslContextFactory.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2020 The Apache Software Foundation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.activemq.artemis.core.remoting.impl.ssl;
+
+import io.netty.handler.ssl.SslContext;
+import java.util.Map;
+import org.apache.activemq.artemis.api.config.ActiveMQDefaultConfiguration;
+import org.apache.activemq.artemis.core.remoting.impl.netty.TransportConstants;
+import org.apache.activemq.artemis.utils.ConfigurationHelper;
+
+/**
+ * Simple class to build a SslContext from a configuration.
+ */
+public class SslContextFactory {
```

Comment:
This class doesn't appear to be used anywhere? If it's not actually used anywhere it should be removed. Otherwise it should probably have a different name since there is already a class named `SSLContextFactory` with different capitalization and in a different package.

Reply:
the goal was to provide netty SslContext but this is not required